appeals/web/src/server/app/app.router.js

```diff
@@ -40,15 +41,20 @@ router.route('/auth/signout').get(asyncRoute(handleSignout));
 
 router
 	.route('/documents/:caseId/upload')
-	.post(assertGroupAccess(config.referenceData.appeals.caseOfficerGroupId), postDocumentsUpload);
+	.post(
+		assertGroupAccess(config.referenceData.appeals.caseOfficerGroupId),
+		addApiClientToRequest,
```

Comment:
I assume we could put this at an `app` or `router` level so it gets applied to all, without changing every route.

That said, might want it to only apply to certain requests (though could use a router.all and filter by method or something).

Reply:
yeah originally was injecting in the appeals routes only, but I thought it makes sense to make it available to every request

Reply:
Yeah makes sense, so you can apply it at the app/router level then

Reply:
`apiClient` is pretty generic and used for all API requests so perhaps this could be done as Ben suggested so it doesn't need to be injected into each route?